tests/src/main/java/net/neoforged/neoforge/unittest/ModConfigSpecTest.java

```diff
@@ -1,163 +0,0 @@
-/*
- * Copyright (c) Forge Development LLC and contributors
- * SPDX-License-Identifier: LGPL-2.1-only
- */
-
-package net.neoforged.neoforge.unittest;
-
-import com.electronwill.nightconfig.core.file.CommentedFileConfig;
-import com.electronwill.nightconfig.core.io.WritingMode;
-import com.google.common.base.Stopwatch;
-import com.google.common.collect.Lists;
-import java.io.File;
-import java.io.IOException;
-import java.lang.reflect.Field;
-import java.nio.file.Files;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Locale;
-import java.util.concurrent.TimeUnit;
-import net.neoforged.neoforge.common.ModConfigSpec;
-import org.junit.Assert;
-import org.junit.Test;
-
-public class ModConfigSpecTest {
```

Comment:
Shouldn't we keep these legacy tests anyway?

Reply:
This test class isn't even testing something, it's abusing junit as performance tests using sout. And I don't think the table thing is even used anymore. I'd say we remove them and add backs the tests testing their functionality if we ever need to.

Reply:
Fair enough.